tests/functional/encryption/test_encryption_configuration_dashboard.py

```diff
@@ -81,7 +81,7 @@ def test_file_block_encryption_configuration_dashboard(self, setup_ui_class):
         """
 
         # Navigate to the block and file page
-        block_and_file_page = PageNavigator().nav_storage_cluster_default_page()
+        block_and_file_page = EncryptionModule().nav_storage_cluster_default_page()
```

Comment:
Reason why PageNavigator is the first Navigation element is because user sees it first when opens the console. This is also the reason why most pages inherit from PageNavigator. This is because it always visible on the left side of the Page. 

nav_storage_cluster_default_page is a method of PageNavigator. Please do not change this flow. All tests must start from PageNavigator.

Reply:
both encryption configuration tests passed: https://jenkins-csb-odf-qe-ocs4.dno.corp.redhat.com/job/qe-deploy-fdf-cluster/77/testReport/